stories/components/Field.js

```diff
@@ -62,5 +62,42 @@ storiesOf('Components', module)
       >
         <Field isReadOnly fieldId="987" />
       </StoryItem>
+
+      <StoryItem
+        title="Field (Editing)"
+        description="A field component when in an editing state"
+      >
+        <Field
+          fieldId="123"
+          label="Field label text"
+          instructions="Instruction text to help inform the user of what they are mean't to add into this field."
+          isStructureEditor
+         >
+          <div>Some dummy text
```

Comment:
What would normally go into here? I imagined something like Facebook text blocks to illustrate dummy content. Do we have those image assets? @nhqe we might need something from you here dude 👍